.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,121 @@
+name: 🐛 Bug Report
+description: Report a reproducible bug or regression.
+title: "[BUG]: "
+labels:
+  - requires triage
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |2-
+          Have a question?  This is the TypeORM issue tracker – and not the right place for general support or questions. Instead, check the ["Support" Documentation](  https://github.com/typeorm/typeorm/blob/master/docs/support.md
+        ) on the best places to ask questions!
+  - type: input
+    id: description
+    attributes:
+      label: Issue description
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: "  A clear and concise description of what you expected to happen."
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Behavior
+      description: >-
+        A clear and concise description of what actually happened. Please wrap
+        any error messages or output in code tags, instead of images.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      description: >-
+        Your bug will be investigated much faster if we can run your code and it
+        doesn't have dependencies other than TypeORM. Issues without
+        reproduction steps or code examples may be closed as not actionable.
+      placeholder: >-
+        Please try to provide a Minimal, Complete, and Verifiable example.
+        http://stackoverflow.com/help/mcve
+    validations:
+      required: true
+  - type: textarea
+    id: env
+    attributes:
+      label: My Environment
+      description: >-
+        Please add any other relevant dependencies to this table at the end. For
+        example: Electron, React Native, or NestJS.
+      value: |
+        | Dependency          | Version  |
+        | ---                 | ---      |
+        | Operating System    |          |
+        | Node.js version     | x.y.zzz  |
+        | Typescript version  | x.y.zzz  |
+        | TypeORM version     | x.y.zzz  |
+    validations:
+      required: true
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Context
+      description: >-
+        Add any other context about the bug report here. Add any other context
+        about the bug report here.
+  - type: textarea
```

Comment:
better to use [checkboxes](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#checkboxes) instead.

Reply:
For this? I don't understand how checkboxes would be used for an additional description...

Reply:
no, I mean to use `checkboxes` instead of `textarea` to display list of databases.